### DISCUSSION TITLE
Meeting 2024-11-05

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#803] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-11-05)

## Announcements:
* cortex-m-rt 0.7.5 released

**Comment by tdittr** [comment]
I did some more work on [drive-rs](https://tweedegolf.github.io/drive-rs/). It now allows you to filter by functionality and also has free-text search.
Not all drivers from the awesome embedded rust list have been ported over yet with all data (like datasheet link), but I'm slowly porting them. However help is appreciated.

If you maintain a driver that is not yet listed feel free to send a PR, all you need to do is fill out a `toml` file with some metadata.

Interesting discovery: When I started on this project there were 6 drivers on the list supporting e-h 1.0. We are now up to 49 drivers on the list supporting e-h 1.0.

**Reply by burrbull** [reply]
It makes sense to add link on your project at the start of https://github.com/rust-embedded/awesome-embedded-rust?tab=readme-ov-file#driver-crates

**Reply by tdittr** [reply]
Sounds like a good idea, I created a PR there now: https://github.com/rust-embedded/awesome-embedded-rust/pull/465

**Reply by andelf** [reply]
Another useful tag might be `async`, which means embassy-compatible.(e-*-async)
I have a repo for drivers: <https://github.com/embedded-drivers/embedded-drivers>. Considering the unmaintained old crates, I use `edrv-` as prefix.

**Reply by tdittr** [reply]
Hey, that should already work, by filtering on dependencies. But I guess an extra filter could make this nicer.

Please feel free to add your drivers :)

**Comment by jannic** [comment]
Perhaps mention https://github.com/rust-lang/rust/issues/130988?
